src/Sarif.UnitTests/TestData/SarifCurrentToVersionOneVisitor/v1/OneRunWithBasicInvocation.sarif

```diff
@@ -27,9 +27,7 @@
         }
       },
       "files": {
-        "collections.rsp": {
-          "uri": "collections.rsp"
-        }
+        "collections.rsp": {}
```

Comment:
The transformer now refrains from emitting `uri` if it matches the key, as permitted by the v1 spec. We could go further and omit the whole object if it's empty but that's more work that I'm willing to do, and would cause problems if the empty object were in a nested file chain.